wasabi/wasabi.go

```diff
@@ -266,3 +266,12 @@ Note: for some providers, UploadFolder requires that a default bucket be set in
 func (w *WasabiCloudStorage) UploadFolder(foldFace interface{}) ([]*types.UploadedFile, error) {
 	return nil, nil
 }
+
+/*
+DeleteFile deletes a file from a bucket on Google Cloud Storage and returns an error if one occurs.
+
+Note: DeleteFile requires that a default bucket be set in bifrost.BridgeConfig.
+*/
```

Comment:
/*
DeleteFile deletes a file Wasabi and returns an error if one occurs.

Note: DeleteFile requires that a default bucket be set in bifrost.BridgeConfig.
*/